noxfile.py

```diff
@@ -173,41 +173,6 @@ def prepare_venv(session: nox.sessions.Session) -> None:
         )
 
 
-@nox.session
```

Comment:
Would now be a good moment to put a `# TODO: ...` saying that Nox's days are numbered?

Reply:
See https://github.com/SciTools/iris/discussions/4806